internal/pkg/selinux/policy/selinux/services/cri.cil

```diff
@@ -60,6 +86,29 @@
 (call protected_f (kube_scheduler_secret_t))
 (allow kube_scheduler_secret_t tmpfs_t (filesystem (associate)))
 
+; Allow kubernetes pods to access their configuration and secrets
+; FIXME: insecure as anyone with access to the pod containerd may obtain these domains
+; TODO: label static pods with the correct domains
+(allow pod_p kube_apiserver_config_t (fs_classes (ro)))
+(allow pod_p kube_apiserver_secret_t (fs_classes (ro)))
+(allow pod_p kube_controller_manager_secret_t (fs_classes (ro)))
+(allow pod_p kube_scheduler_config_t (fs_classes (ro)))
```

Comment:
As static pods aren't given any special security status, we have to allow this to all pods, however these are privileged to kube pods